themes-default/slim/static/js/templates/config.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div id="config-content">
```

Comment:
How do you feel about:
```vue
    <div v-if="!config.pythonVersion" class="vue-loading" id="config-content">
        <h3>Loading...</h3>
    </div>
    <div v-else id="config-content">
```

It gives it a cleaner look in the meantime, but a little more work when it's no longer needed.
I added `class="vue-loading"` so if we keep it the same on all `.vue` files it would be easy to remove them.

Reply:
I was thinking we could just add a loading component before the `<router-view>`?

```
<loading v-if="loading"></loading>
<router-view v-else></router-view>

Reply:
How would you know when the values were loaded?

Reply:
Well for example on this you'd set `loading: true` on data and then set a function to watch `config` when it updates set `loading: false`.